.circleci/config.yml

```diff
@@ -0,0 +1,45 @@
+version: 2.1
+
+jobs:
+  tests-py36:
+    docker:
+      - image: 'cimg/python:3.6'
+    steps:
+      - checkout
+      - run: pip install --user tox
+        # I didn't find a way using TOXENV with regular expression, similar to TOX_SKIP_ENV
+        # https://tox.readthedocs.io/en/latest/config.html#conf-envlist
+      - run: tox -e $(tox -l | grep -e "^py36" | paste -sd ",")
```

Comment:
I don't really understand this comment. It should explain what this function is doing I think. Is it basically just running `tox -e py36*`

Reply:
More discussion on that here https://github.com/tox-dev/tox/issues/238 (worth mentioning the issue in the comment)

Reply:
> Is it basically just running `tox -e py36*`

Yes. That's what it does. However, that exact command you wrote doesn't work 😄 . So, this hack is a way to do exactly what we expect when running `tox -e py36*`

_The other_ way to achieve the same behavior is using `TOX_SKIP_ENV` with a regex that negates the matching of the envs we want.

I'm not strong on one workaround or the other. Both work. I just found the one I wrote here first.

Reply:
I used `TOX_SKIP_ENV` as suggested in the issue.

Reply:
You can just also make this explicit like this https://github.com/readthedocs/readthedocs-sphinx-search/blob/d857859f851b210e7b2658d9efdc25c8882c0f41/.circleci/config.yml#L21

Reply:
I don't care too much about the solution as long as it works. I just want to know what its doing in the comment, which worked well 👍